packages/client/modules/userDashboard/components/OrgAuthentication/OrgAuthenticationMetadata.tsx

```diff
@@ -0,0 +1,98 @@
+import styled from '@emotion/styled'
+import React, {useState} from 'react'
+import makeMaxWidthMediaQuery from '~/utils/makeMaxWidthMediaQuery'
+import DialogTitle from '../../../../components/DialogTitle'
+import BasicTextArea from '../../../../components/InputField/BasicTextArea'
+import SecondaryButton from '../../../../components/SecondaryButton'
+import {PALETTE} from '../../../../styles/paletteV3'
+
+const mobileBreakpoint = makeMaxWidthMediaQuery(420)
+const largeMobileBreakpoint = makeMaxWidthMediaQuery(540)
+
+const Section = styled('div')({
+  margin: '0 auto',
+  maxWidth: '608px',
+  padding: '0px 16px 12px 16px',
```

Comment:
As suggested above, we can just set fixed paddings we need `padding: 24px 16px 28px 34px;` and don't use margin and maxWidth